src/main/java/org/cascadebot/cascadebot/music/KaraokeHandler.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2020 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package org.cascadebot.cascadebot.music;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.TextChannel;
+import okhttp3.Call;
+import okhttp3.Callback;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.apache.commons.lang.StringEscapeUtils;
+import org.apache.tools.ant.filters.StringInputStream;
+import org.cascadebot.cascadebot.CascadeBot;
+import org.cascadebot.cascadebot.MDCException;
+import org.cascadebot.cascadebot.data.language.Language;
+import org.cascadebot.cascadebot.messaging.MessageType;
+import org.cascadebot.cascadebot.messaging.Messaging;
+import org.jetbrains.annotations.NotNull;
+import org.w3c.dom.Document;
+import org.w3c.dom.NamedNodeMap;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.IOException;
+import java.util.*;
+
+public class KaraokeHandler {
+
+    private static Set<Long> karaokeEnabled = new HashSet<>();
+
+    public static boolean isKaraoke(Long guildId) {
+        if (!karaokeEnabled.contains(guildId)) {
+            return false;
+        }
+        return karaokeEnabled.contains(guildId);
+    }
+
+    public static void setKaraoke(Long guildId, Boolean status) {
+        if (status) {
+            karaokeEnabled.add(guildId);
+        } else {
+            karaokeEnabled.remove(guildId);
+        }
+    }
+
+    public static void getSongLyrics(String trackId, TextChannel channel, Long guildId, Message message) throws ParserConfigurationException {
```

Comment:
Catch the ParserConfiguratioException and use sendException to log it, it shouldn't happen